Sources/BugsnagPerformance/Private/ResourceAttributes.mm

```diff
@@ -162,7 +162,7 @@
 ResourceAttributes::get() noexcept {
     auto infoDictionary = NSBundle.mainBundle.infoDictionary;
     return @{
-        @"bugsnag.app.bundle_version": infoDictionary[@"CFBundleVersion"] ?: [NSNull null],
+        @"bugsnag.app.bundle_version": (id)configuration_.bundleVersion ?: infoDictionary[@"CFBundleVersion"] ?: [NSNull null],
```

Comment:
For other config options, we set a default value when the configuration object is created, if it hasn't been provided. Any reason not to do this here so that the config object always contains the value in case it's used elsewhere in future?

(This would also match the notifier implementation.)